cloudbuild.yaml

```diff
@@ -47,209 +47,98 @@
 #    https://github.com/GoogleCloudPlatform/cloud-builders/tree/master/go
 
 steps:
-- id: 'Docker Image: open-match-base-build'
+- id: 'Docker Image: open-match-build'
   name: gcr.io/cloud-builders/docker
-  args: ['build', '-t', 'open-match-base-build', '-f', 'Dockerfile.base-build', '.']
+  args: ['build', '--build-arg', 'GCP_PROJECT_ID=$PROJECT_ID', '-t', 'open-match-build', '-f', 'Dockerfile', '.']
```

Comment:
Correct me if I am wrong :) Could we use something like docker build -t open-match-build - < Dockerfile since it seems like the base build does not requires any context?